pkg/messaging/v1/invoke_method_response.go

```diff
@@ -70,10 +69,8 @@ func (imr *InvokeMethodResponse) WithHeaders(headers metadata.MD) *InvokeMethodR
 func (imr *InvokeMethodResponse) WithFastHTTPHeaders(header *fasthttp.ResponseHeader) *InvokeMethodResponse {
 	var md = DaprInternalMetadata{}
 	header.VisitAll(func(key []byte, value []byte) {
-		md[string(key)] = &structpb.ListValue{
-			Values: []*structpb.Value{
-				{Kind: &structpb.Value_StringValue{StringValue: string(value)}},
-			},
+		md[string(key)] = &internalv1pb.ListStringValue{
```

Comment:
Does this have to be a pointer?

Reply:
Yes, this has to be a pointer. `map<string, ListStringValue>` in Protobuf file is generated to `map[string]*ListStringValue` in go. this is just helper.

Reply:
@yaron2 please review this again. no way to remove pointer completely even if we use map<string, ListValue>